ui/packages/clinic-client/swagger/spec.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
This means we need to sync the clinic or cloud api specs from those projects. If we still need to maintain the multi projects in separate repos, why don't we put the dashboard-for-x codes in the `x` repository?